editors/document-model/editor-initital-state.tsx

```diff
@@ -65,9 +65,18 @@ export default function EditorInitialState({
                 onChange={value => setCode(value ?? '')}
                 {...props}
                 value={code}
-                onMount={(editor, monaco) => {
+                onMount={async (editor, monaco) => {
                     editorRef.current = editor;
                     props.onMount?.(editor, monaco);
+                    setTimeout(async () => {
```

Comment:
without this timeout, format is not applied: https://github.com/react-monaco-editor/react-monaco-editor/issues/89